crates/turbopack-ecmascript/tests/tree-shaker/analyzer/failed-1/output.md

```diff
@@ -37,8 +37,9 @@ function getSocketProtocol(assetPrefix) {
 
 - Hoisted
 - Declares: `getSocketProtocol`
-- Reads (eventual): `location`, `URL`
+- Reads (eventual): `URL`
```

Comment:
Also reads to externals vars doesn't need to be tracked

Reply:
I think we should ignore them. Otherwise module evaluation would depend on everything.

writes to external vars should be tracked as `side effects`. `read`/`write` should only track local vars/bindings.